statusHandler/presenter/common.go

```diff
@@ -30,3 +35,45 @@ func (psh *PresenterStatusHandler) getFromCacheAsString(metric string) string {
 
 	return valStr
 }
+
+func plural(count int, singular string) (result string) {
+	if (count == 1) || (count == 0) {
+		result = strconv.Itoa(count) + " " + singular + " "
+	} else {
+		result = strconv.Itoa(count) + " " + singular + "s "
+	}
+	return
+}
+
+func secondsToHuman(input int) (result string) {
+	years := math.Floor(float64(input) / 60 / 60 / 24 / 7 / 30 / 12)
```

Comment:
refactor this code with less magic numbers.

Reply:
Done.